### DISCUSSION TITLE
Analytics more than 1 month

### DISCUSSION THREAD

**Discussion by alexandre-mace** [vercel/community#2589] [Help] [answered]
### Summary

I have a React project deployed with a free tier account, i used my 1 allowed analytics for this specific project. 

It has been deployed 4 month ago, analytics was working properly with 3 month old analytics logs. 

Now i can only see the analytics from 1 month ago idk why, i would maybe like to upgrade as i understood getting the whole history seems to be a paid option.

My need is simple just see how many users are on my website without using google analytics.

So here are my two questions : 
Where is the 3 month old analytics logs gone instead of the only 1 month old option displayed in the interface?
What would be the cost of obtaining all the logs necessary to determine the total number of users for this basic requirement?

Thanks in advance

**Comment by amyegan** [comment, marked as answer]
Hi @alexandre-mace. While Vercel Web Analytics was in beta, there was a 3 month limit to let people give it a try. Now the limit is 30 days for Hobby accounts and 12 months for Pro teams. You can find a table comparing features and limits across plan types here: https://vercel.com/docs/concepts/analytics/limits-and-pricing
